File: API/Helpers/UserParams.cs
namespace API.Helpers
{
    public class UserParams:PaginationParams
    {
        
        public string CurrentUsername { get; set; }
        public string Gender { get; set; }
        public int SchoolId { get; set; }
        public int ClassId { get; set; }
        public string City { get; set; }
        public string OrderBy { get; set; } = "lastActive";

    }
}

File: Social/src/app/_models/userParams.ts
import { User } from './user';

export class UserParams {
  schoolId!: number;
  classId!: number;
  city!: string;
  pageNumber = 1;
  PageSize = 12;
  orderBy="lastActive"

  constructor(schoolId: number, classId: number, city: string){
    this.schoolId=schoolId;
    this.classId=classId;
    this.city=city;
  }
}
